packages/renderer/src/lib/appearance/appearance-util.ts

```diff
@@ -17,25 +17,36 @@
  ***********************************************************************/
 
 import { AppearanceSettings } from '../../../../main/src/plugin/appearance-settings';
+import { configurationProperties } from '../../stores/configurationProperties';
 
 export class AppearanceUtil {
-  async isDarkMode(): Promise<boolean> {
-    // get the configuration of the appearance
-    const appearance = await window.getConfigurationValue<string>(
-      AppearanceSettings.SectionName + '.' + AppearanceSettings.Appearance,
-    );
+  isDark: boolean = false;
```

Comment:
it needs to be reactive (state/ or writable store)

Reply:
The `$state` rune is only available inside `.svelte` and `.svelte.js/ts` files

Reply:
appearance-util.ts can be renamed to appearance.util.svelte.ts (it's on the client side and imported only in svelte file)
you can also use a Writable (or Readable store)